hw/ip/otbn/README.md

```diff
@@ -13,11 +13,37 @@ through the [GitHub issue tracker](https://github.com/lowRISC/opentitan/issues).
 
 ## Develop OTBN
 
-### Build the RTL implementation
+### Run the standalone simulation
 
-To build the RTL implementation of OTBN in a simulation, run `fusesoc` without
-passing the `OTBN_MODEL` flag. For example, the Verilator simulation can be
-compiled as follows.
+*Note that OTBN is still in the early development stages so this simulation will
+do very little currently*
+
+A standalone environment to run OTBN alone in verilator is included. It is built
+with `fusesoc`:
```

Comment:
Nit: The "is" in the two sentences mean different things, which looks a bit odd. Probably easiest to change the second sentence to an instruction: "Build it with `fusesoc` as follows:"